### PR TITLE
Bump yarn version, network-timeout to fix ESOCKETTIMEDOUT network issue in CI

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

File: Dockerfile.builder [hunk 1]
```diff
@@ -17,7 +17,7 @@ RUN go install github.com/jstemmer/go-junit-report@latest
 
 ### Install NodeJS and yarn
 ENV NODE_VERSION="v14.16.0"
-ENV YARN_VERSION="v1.22.10"
+ENV YARN_VERSION="v1.22.19"
```
Comment (spadgett) [inline Dockerfile.builder:20]: Hey @rhamilto it's actually using the yarn version checked into the repo here:

https://github.com/openshift/console/tree/master/frontend/.yarn/releases

See the README here for steps to update:

https://github.com/openshift/console#frontend

Comment (rhamilto) [reply Dockerfile.builder]: Thanks, @spadgett.  Updated.  🤞

**Review by spadgett — APPROVED** [review]
/lgtm

Thanks, @rhamilto. This is a good change to make regardless of whether it addresses the timeout issue.

**Comment by rhamilto** [comment]
Sigh.  This didn't fix the ESOCKETTIMEDOUT error.  https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_console/11801/pull-ci-openshift-console-master-e2e-gcp-console/1545453534219079680/build-log.txt

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request closed without merging**